Set 1: Basic Arithmetic

1.Calculate the sum of 123 and 456.
2.Find the difference between 789 and 321.
3.Calculate the product of 15 and 25.
4.Find the quotient when 100 is divided by 5.
5.Calculate the remainder when 47 is divided by 9.

In [1]:
print(123+456)
print(789-321)
print(15*25)
print(100/5)
print(47%9)

579
468
375
20.0
2


Set 2: Exponents and Roots

1.Calculate 3 raised to the power of 4.
2.Find the square root of 225.
3.Calculate the cube root of 64.
4.Calculate 2 to the power of 10.
5.Find the square root of 300 (you can use a library function or approximate).

In [1]:
print(3**4)
print(225**(1/2))
print(64**(1/3))
print(2**10)
print(300**(1/2))


81
15.0
3.9999999999999996
1024
17.320508075688775


Set 3: Order of Operations (PEMDAS/BODMAS)

1.Evaluate: 10 + 5 * 2
2.Evaluate: (10 + 5) * 2
3.Evaluate: 20 - 10 / 2 + 5
4.Evaluate: 16 / 4 * 2 - 1
5.Evaluate: 3 * (4 + 2) / 2

In [6]:
print(10+5*2)
print((10 + 5) * 2)
print(20 - 10 / 2 + 5)
print(16 / 4 * 2 - 1)
print(3 * (4 + 2) / 2)


20
30
20.0
7.0
9.0


Set 4: Working with Decimals

1.Calculate the sum of 3.14 and 2.71.
2.Find the product of 1.5 and 2.5.
3.Divide 10.0 by 3.0.
4.Calculate 2.5  to the power of 2.
5.Find the square root of 6.25.

In [7]:
a=float(input())
b=float(input())
x=a+b
print(x)


5.85


In [8]:
a=float(input())
b=float(input())
x=a*b
print(x)

3.75


In [9]:
a=float(input())
b=float(input())
x=a/b
print(x)

3.3333333333333335


In [10]:
a=float(input())
b=float(input())
x=a**b
print(x)

6.25


In [11]:
a=float(input())
x=a**(1/2)
print(x)

2.5


Set 5: Mixed Operations

1.Evaluate: 5 * 2 + 10 / 2 - 3
2.Evaluate: (8 + 4) / 2 * 3 - 1
3.Evaluate: 15 - 3 * (2 + 1) + 7
4.Evaluate: 20 / 4 + 2 * 5 - 6
5.Evaluate: 4 * (6 - 2) / 8 + 3

In [12]:
print(5 * 2 + 10 / 2 - 3)
print((8 + 4) / 2 * 3 - 1)
print(15 - 3 * (2 + 1) + 7)
print(20 / 4 + 2 * 5 - 6)
print(4 * (6 - 2) / 8 + 3)

12.0
17.0
13
9.0
5.0


Set 6: More Practice

1.Calculate the area of a triangle with base 10 and height 5. (Area = 0.5 * base * height)
2.Calculate the volume of a sphere with radius 4. (Volume = (4/3) * pi * radius^3. Use pi ≈ 3.14159)
3.Convert 100 degrees Celsius to Fahrenheit. (Fahrenheit = (Celsius * 9/5) + 32)
4.Convert 68 degrees Fahrenheit to Celsius.
5.Calculate the circumference of a circle with diameter 12. (Circumference = pi * diameter)

In [13]:
b=float(input())
h=float(input())
a=1/2*b*h
print(a)

25.0


In [14]:
r=float(input())
v=4/3*3.14159*(r**3)
print(v)


268.08234666666664


In [15]:
c=float(input())
a=(c*9/5)+32
print(a)

212.0


In [16]:
f=float(input())
a=(f-32)/(9/5)
print(a)

20.0


In [17]:
d=float(input())
a=3.14*d
print(a)

37.68


Set 7: Integer Division and Modulus

1.Find the quotient and remainder when 17 is divided by 3. (Use the / and % operators)
2.Find the quotient and remainder when 25 is divided by 6.
3.Is 12 divisible by 4? (Use the modulus operator)
4.Is 15 divisible by 2?
5.What is the result of 10 // 3 (floor division)?

In [33]:
a=float(input())
b=float(input())
x=a/b
y=a%b
print("quotient",x,"\n","remainder", y)

quotient 5.666666666666667 
 remainder 2.0


In [23]:
a=float(input())
b=float(input())
x=a%b
print(x)

1.0


In [24]:
print(10//3)

3


Set 8:  Challenging Problems

1.A rectangle has a length of 8 and a width of 5. Calculate its area and perimeter.(area=length*width, perimeter= 2*(length+width))
2.A circle has a radius of 7. Calculate its area and circumference.(area=3.14*radius square , circuference = 2*3.14*radius)
3.A car travels at a speed of 60 km/h. How far will it travel in 2.5 hours?(distance=speed*time)
4.A store offers a 20% discount on all items. If an item costs $50, what is the discounted price?(price -(p*(discount/100)))
5.Calculate the average of 10, 20, 30, 40, and 50.(sum of all no./count of all no.      eg: 10+20+30/3)

In [30]:
l=float(input())
b=float(input())
a=l*b
p=2*(l+b)
print("area = ", a, "perimeter = ", p)

area =  40.0 perimeter =  26.0


In [31]:
r=float(input())
a=3.14*(r**2)
c=2*r*3.14
print("area  =",a, "circuference =", c)

area  = 153.86 circuference = 43.96


In [36]:
s=float(input())
t=float(input())
d=s*t
print(d)

150.0


In [38]:
p=float(input())
d=float(input())
dp=p-(p*(d/100))
print(dp)

40.0


In [39]:
a=float(input())
b=float(input())
c=float(input())
d=float(input())
e=float(input())
x=(a+b+c+d+e)/5
print(x)

30.0


Set 9:  More Challenges

1.A train travels 360 km in 4 hours. What is its average speed?(avg speed= distance/time)
2.A shopkeeper buys 20 apples for $1 each and sells them for $1.5 each. What is his profit?(profit =selling cost =buying cost )
3.Calculate the simple interest on a principal of $1000 at a rate of 5% per annum for 3 years. (Simple Interest = Principal * Rate * Time)
4.Convert 2.5 meters to centimeters. (1 meter = 100 centimeters)
5.Convert 1.5 kilograms to grams. (1 kilogram = 1000 grams)

In [40]:
d=float(input())
t=float(input())
s=d/t
print(s)

90.0


In [41]:
am=float(input("amount"))
bc=float(input("buying price"))
sp=float(input("selling price"))
tbc=am*bc
tsp=sp*am
p=tsp-tbc
print(p)

10.0


In [43]:
p=float(input())
r=float(input())
t=float(input())
si=(p*r*t)/100
print(si)

150.0


In [44]:
m=float(input())
cm=m*100
print(cm)

250.0


In [45]:
km=float(input())
g=km*1000
print(g)

1500.0


Set 10:  Final Set

1.Calculate the area of a trapezoid with bases 10 and 12 and height 6. (Area = 0.5 * (base1 + base2) * height)
2.Calculate the volume of a cylinder with radius 5 and height 10. (Volume = pi * radius^2 * height)
3.A person invests $5000 at an interest rate of 4% compounded annually. How much will they have after 2 years? (Use the formula for compound interest)
4.Calculate the distance between two points (x1, y1) and (x2, y2) in a 2D plane. (Distance = sqrt((x2 - x1)^2 + (y2 - y1)^2)) Assume the points are (1, 2) and (4, 6).
5.A ball is thrown upwards with an initial velocity of 20 m/s. Calculate the maximum height it reaches. (Use the formula: h = (v^2) / (2 * g), where v is the initial velocity and g is the acceleration due to gravity, approximately 9.8 m/s^2)

In [46]:
b1=float(input("base2"))
b2=float(input("base1"))
h=float(input("hight"))
a=0.5*(b1+b2)*h
print(a)

66.0


In [47]:
r=float(input("radius"))
h=float(input("hight"))
v=3.14*(r**2)*h
print(v)

785.0


In [2]:
p=float(input("price"))
r=float(input("rate"))
t=float(input("Time"))
n=float(input("no. of term"))
a= p *(1 + (r/100)/n)**(n*t)
print(a)

5408.000000000001


In [48]:
x1=float(input("x1"))
x2=float(input("x2"))
y1=float(input("y1"))
y2=float(input("Y2"))
d=((x2-x1)**2+(y2-y1)**2)**(1/2)
print(d)

5.0


In [49]:
v=float(input("velocity"))
h=(v**2) / (2 * 9.8)
print(h)

20.408163265306122


Set 2: Word Problems - Distance, Speed, Time (10 Questions)

A car travels at 60 km/h for 2.5 hours. How far does it travel?
A train travels 420 km in 7 hours. What is its average speed?
A plane flies at 800 km/h for 1.5 hours. How far does it fly?
A cyclist rides at 20 km/h for 3 hours. How far does the cyclist travel?
A boat sails at 15 knots for 4 hours. How far does the boat travel? (1 knot ≈ 1.852 km/h)
A car travels 300 km at an average speed of 75 km/h. How long does the journey take?
A train travels 560 km at an average speed of 80 km/h. How long does the journey take?
A plane flies 1200 km at an average speed of 600 km/h. How long does the flight take?
A cyclist rides 45 km at an average speed of 15 km/h. How long does the ride take?
A boat sails 60 nautical miles at an average speed of 10 knots. How long does the journey take?
Set 3: Word Problems - Cost and Profit (10 Questions)
output:

150 km
60 km/h
1200 km
60 km
111.12 km (approximately)
4 hours
7 hours
2 hours
3 hours
6 hours


In [4]:
d=float(input("distance"))
t=float(input("time"))
speed=d/t
print("speed = ", speed, "km/h")

speed =  60.0


In [5]:
s=float(input("speed"))
t=float(input("time"))
distance=s*t
print("distance = ", distance, "km")

distance =  1200.0 km


In [ ]:
s=float(input("speed"))
d=float(input("distance"))
time=d/s
print("time = ", time, "hours")


Set 3: Word Problems - Cost and Profit
1.A shopkeeper buys 40 apples for $0.5 each and sells them for $0.75 each. What is his profit?
2.A store sells oranges for $0.6 each. If you buy 12 oranges, what is the total cost?
3.A person buys 3 shirts for $15 each and 2 pairs of pants for $25 each. What is the total cost?
4.A baker makes 24 cakes at a cost of $3 each and sells them for $5 each. What is his profit?
5.A farmer sells 50 kg of potatoes for $0.8 per kg. How much money does he make?
6.A shopkeeper buys 60 pens for $0.25 each and sells them for $0.5 each. What is his profit?
7.A store sells notebooks for $1.5 each. If you buy 8 notebooks, what is the total cost?
8.A person buys 4 books for $12 each and 3 DVDs for $18 each. What is the total cost?
9.A fruit seller buys 80 mangoes for $0.4 each and sells them for $0.7 each. What is his profit?
10.A store sells t-shirts for $10 each. If you buy 5 t-shirts, what is the total cost?

output:
Set 3: Word Problems - Cost and Profit

$10
$7.2
$95
$48
$40
$15
$12
$102
$24
$50

In [13]:
am=float(input("amount"))
bc=float(input("buying price"))
sc=float(input("selling price"))
tbc=am*bc
tsc=am*sc
p=tsc-tbc
print("profit = ", p)

profit =  24.0


In [10]:
am=float(input("amount"))
bc=float(input("buying price"))
tbc=am*bc
print("total cost price" , tbc)

total cost price 12.0


In [12]:
ams=float(input("amount of shirt"))
amp=float(input("amount of pant"))
bcs=float(input("buying price of shirt"))
bcp=float(input("buying price of pant"))
tbcs=ams*bcs
tbcp=amp*bcp
total=tbcs+tbcp
print("total cost = ", total)

total cost =  102.0


Set 4: Word Problems - Area, Perimeter, Volume (10 Questions)

1.A rectangle has a length of 12 cm and a width of 8 cm. What is its area and perimeter?
2.A square has a side length of 7 meters. What is its area and perimeter?
3.A circle has a radius of 6 cm. What is its area and circumference? (use π ≈ 3.14)
4.A triangle has a base of 10 cm and a height of 8 cm. What is its area?
5.A cube has a side length of 5 meters. What is its volume?
6.A rectangular prism has a length of 8 cm, a width of 4 cm, and a height of 3 cm. What is its volume?
7.A sphere has a radius of 4 cm. What is its volume? (use π ≈ 3.14)
8.A cylinder has a radius of 3 cm and a height of 10 cm. What is its volume? (use π ≈ 3.14)
9.A cone has a radius of 5 cm and a height of 12 cm. What is its volume? (use π ≈ 3.14) (Volume of cone = (1/3)πr²h)
10.A pyramid has a square base with side length 6 cm and a height of 8 cm. What is its volume? (Volume of pyramid = (1/3) * base area * height)

output:
Set 4: Word Problems - Area, Perimeter, Volume

Area: 96 cm², Perimeter: 40 cm
Area: 49 m², Perimeter: 28 m
Area: 113.04 cm², Circumference: 37.68 cm
40 cm²
125 m³
96 cm³
267.94 cm³ (approximately)
282.6 cm³ (approximately)
314.16 cm³ (approximately)
72 cm³

rectangle,area and perimeter

In [2]:
l=float(input("length"))
w=float(input("width"))
a=l*w
p=2*(l+w)
print("area = ", a,"cm²" "\n","perimeter = ", p,"cm")

area =  96.0 cm²
 perimeter =  40.0 cm


square ,area and perimeter

In [3]:
s=float(input("length "))
P=4*s
a=s**2
print("area = ", a,"m²",   "\n","perimeter = ", P,"m")

area =  49.0 cm² 
 perimeter =  28.0 cm


circle,area and circumference

In [4]:
r= float(input("radius"))
a=3.14*(r**2)
c=2*3.14*r
print("area = ",a,"cm²","\n","circumference = ",c,"cm")

area =  113.04 cm² 
 circumference =  37.68 cm


triangle,area 

In [6]:
b=float(input("base"))
h=float(input("hight"))
a=0.5*b*h
print("area = ",a,"cm²")

area =  40.0 cm²


cube,volume

In [7]:
s=float(input("side"))
v=s**3
print("volume = ",v)


volume =  125.0


rectangular prism,volume

In [8]:
l=float(input("length"))
w=float(input("width"))
h=float(input("hight"))
v=l*w*h
print("volume = ",v,"m³")


volume =  96.0 m³


sphere

In [9]:
r=float(input("radius"))
v=(4/3)*3.14*(r**3)
print("volume = ",v,"cm³")


volume =  267.94666666666666 cm³


cylinder,volume =  𝜋r**2h


In [11]:
r=float(input("radius"))
h=float(input("hight"))
v=3.14*(r**2)*h
print("volume = ",v,"cm³")

volume =  282.6 cm³


cone, volume=(1/3)*π*(r**2)*h

In [12]:
r=float(input("radius"))
h=float(input("hight"))
v=(1/3)*3.14*(r**2)*h
print("volume = ",v,"cm³")

volume =  314.0 cm³


pyramid, volume= (1/3)*(s**2)*h

In [16]:
s=float(input("length"))
h=float(input("hight"))
v=(1/3)*(s**2)*h
print("volume = ",v,"cm³")

volume =  96.0 cm³


Set 5: Mixed Word Problems (10 Questions)

1.A person invests $2500 at a simple interest rate of 5% per annum for 3 years. How much interest will they earn?
2.A car travels 360 km in 4 hours. What is its average speed in m/s?
3.A store offers a 15% discount on all items. If an item costs $80, what is the discounted price?
4.A recipe calls for 2 cups of flour, 1 cup of sugar, and 1/2 cup of butter. If you want to make 2.5 times the recipe, how much of each ingredient do you need?
5.A ball is thrown upwards with an initial velocity of 18 m/s. What is the approximate maximum height it reaches? (g ≈ 10 m/s²)
A ladder is 5 meters long and leans against a wall. The base of the ladder is 3 meters away from the wall. How high up the wall does the ladder reach?
A person buys 6 notebooks for $2 each and 4 pens for $1.5 each. What is the total cost?
A rectangular garden is 15 meters long and 10 meters wide. What is its area and perimeter?
A circle has a diameter of 14 cm. What is its circumference and area?
A train travels at 90 km/h for 2 hours and then at 60 km/h for 1.5 hours. What is the total distance traveled?

output:
Set 5: Mixed Word Problems

$375
16.67 m/s (approximately)
$68
Flour: 5 cups, Sugar: 2.5 cups, Butter: 1.25 cups
16.2 meters (approximately)
4 meters
$16
Area: 150 m², Perimeter: 50 m
Circumference: 43.98 cm, Area: 153.94 cm²
270 km

In [1]:
p=float(input("price"))
r=float(input("rate"))
t=float(input("Time"))
si=p*(r/100)*t
print("intrest = ",si)

intrest =  375.0


In [ ]:
c=float(input("cost"))
d=float(input("discount"))
di=(c*d)/100
td=c-di
print("total discount",td)

total discount 68.0


In [7]:
n=float(input("amount of recipe"))
f=n*2
s=n*1
b=n*(1/2)
print(" flout = ",f,"\n","sugger = ",s,"\n","butter = ",b)


 flout =  5.0 
 sugger =  2.5 
 butter =  1.25


In [2]:
x=int(input())
y=int(input())
z=(x*y)+1
for i in range(1,z):
    if i%x==0 and i%y==0:
        print(i)
        break


72


In [10]:
x=int(input())
y=int(input())
if x>y:
    z=y
else:
    z=x
for i in range(z,0,-1):
    if x%i==0 and y%i==0:
        print(i)
        break
    

3
